### ISSUE TITLE
Usage of service

### ISSUE THREAD

**Issue by AngryJKirk** [mgrankin/ru_transformers#4]
Hello!

Superb work, I've been looking for thing like that for quite a long time :)

But I am very new in Python and especially in DL, so I have 2 questions:

1) How to use it to get text output?

2) What is the requirements to model? For example, I have a lot of Russian dialogs, usually not very connected to each other, will it fit well?

**Comment by mgrankin** [comment]
1. Start with looking at the code located in `rest.py`, that's an example.
2. You should try and only when you'll know. Just be careful with overfitting, it's very easy to overfit on small dataset.

**Comment by gpt2ent** [comment]
Do you detect overfitting by looking at samples or something else (perplexity scores, etc)?

**Comment by mgrankin** [comment]
I'm aware of only one way to detect overfitting - by using a validation set.

**Comment by mrcatman** [comment]
Hi, I'm also very new to ML, and I want to ask a few questions:
1) Is there a simple way to train the model on plain TXT files?
2) How big should the model be to give out enough meaningful results?
Thanks in advance.

**Comment by mgrankin** [comment]
1. Yes, the README is all about that.
2. There are two models trained in this repo. Try both to find which one is works for you.
